### ISSUE TITLE
Add interactivity to home page Teams grid

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/wporg-make#15]
The initial implementation simply displays all the team information, the same as the old theme did:

![Image](https://github.com/user-attachments/assets/fc64f257-d834-4f67-aa06-bda0007cb7da)

The design envisioned an interactive grid with expanding cells:

![Image](https://github.com/user-attachments/assets/9b601d96-cb68-4d8b-a4c2-f171d4e29675)

The design has important accessibility concerns; the content should only be visually hidden so that screen readers can still access all the team info. The expand button (caret next to title) should be hidden from screen readers.

**Comment by adamwoodnz** [comment]
Superseded by [new design](https://github.com/WordPress/wporg-make/issues/2#issuecomment-2457839521)
